### PR TITLE
[PHP 8.2] Document mysqli_execute_query

### PR DESCRIPTION
**Pull request by kamil-tekiela**
I hope I haven't made any mistakes. :)

### CODE REVIEW

File: reference/mysqli/mysqli/execute-query.xml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+   Due to this limitation, the status information is copied to the
+   <classname>mysqli</classname> object and is available using its methods, e.g.
+   <function>mysqli_affected_rows</function> or <function>mysqli_error</function>.
+  </para>
+  <note>
+   <para>
+    In the case where you pass a statement to
+    <function>mysqli_execute_query</function> that is longer than
+    <literal>max_allowed_packet</literal> of the server, the returned
+    error codes are different depending on whether you are using Windows or Linux.
```
Comment (cmb69) [inline reference/mysqli/mysqli/execute-query.xml:52]: We should avoid "you"; so maybe:
```suggestion
    In the case where a statement is passed to
    <function>mysqli_execute_query</function> that is longer than
    <literal>max_allowed_packet</literal> of the server, the returned
    error codes are different depending on the operating system.
```

File: reference/mysqli/mysqli/execute-query.xml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        values for a row), or in a comparison with a column in a
+        <literal>WHERE</literal> clause to specify a comparison value.
+       </para>
+       <para>
+        However, they are not permitted for identifiers (such as table or
+        column names), or to specify both
+        operands of a binary operator such as the <literal>=</literal> equal
+        sign. The latter restriction is necessary because it would be
+        impossible to determine the parameter type. In general, parameters are
+        legal only in Data Manipulation Language (DML) statements, and not in
```
Comment (cmb69) [inline reference/mysqli/mysqli/execute-query.xml:104]: I assume parameters are also legal in Data Query Language (DQL) statements.

**Review by cmb69 — APPROVED** [review]
Besides the two issues, this looks good to me. Thank you!

File: reference/mysqli/mysqli/execute-query.xml (hunk 1 above)
Comment (kamil-tekiela) [reply reference/mysqli/mysqli/execute-query.xml:52]: That's copy-pasta.

File: reference/mysqli/mysqli/execute-query.xml (hunk 2 above)
Comment (kamil-tekiela) [reply reference/mysqli/mysqli/execute-query.xml]: I removed it here and in the other two places.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Comment by cmb69** [comment]
@kamil-tekiela, please squash the commits next time (otherwise it get's confusing for translaters).

**Pull request merged**